Week-4/iterator.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Exercise 4.3
+ * 
+ * Implement Fibonacci Series with Iterators
+*/
+
+const Fib = (n) => ({
+
+    [Symbol.iterator]: () => {
+        let i = 1;
+        let thisNum = 0, nextNum = 0;
+        return {
+            next: () => {
+                i++;
+                if(i > n){
+                    return { done: true }
+                }
+                thisNum = nextNum;
+                nextNum = (thisNum + nextNum) || 1;
```

Comment:
```suggestion
          [thisNum, nextNum] = [nextNum, (thisNum + nextNum) || 1];
```

Reply:
fixed in [feb55eb](https://github.com/pesto-students/p8-manish-AkashDPA/pull/5/commits/feb55eb5318a7bcaa785ab2fe5448998c704c6da)